hadoop-hdds/docs/content/design/secure-s3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Secure S3 keys management
+summary: Improving security regarding s3 keys management
+date: 2023-03-10
+jira: HDDS-8132
+status: implemented
```

Comment:
Just clarifying, a patch is available for remote S3 storage but, the implementation of HTTP endpoint for S3 secret retrieval is in progress correct?

Reply:
Should we change the status to implementing then?